src/seedu/addressbook/data/address/Block.java

```diff
@@ -0,0 +1,21 @@
+package seedu.addressbook.data.address;
+
+public class Block {
```

Comment:
Missing header comment. All non trivial classes/methods should have javadoc format header comments.